bin/ovpn_genconfig

```diff
@@ -118,7 +119,7 @@ process_push_config() {
   local ovpn_push_config=''
   ovpn_push_config="$1"
   echo "Processing PUSH Config: '${ovpn_push_config}'"
-  [[ -n "$ovpn_push_config" ]] && echo "push $ovpn_push_config" >> "$TMP_PUSH_CONFIGFILE"
+  [[ -n "$ovpn_push_config" ]] && echo "push $(getroute $ovpn_push_config)" >> "$TMP_PUSH_CONFIGFILE"
```

Comment:
I get what you're trying to do here... but this section of for arbitrary push options, not only route pushes.